seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -450,7 +450,7 @@
       ],
       "tests": [
         "assert(myVar === 10, 'message: <code>myVar</code> should equal <code>10</code>');",
-        "assert(/myVar[-]{2}/.test(code), 'message: Use the <code>--</code> operator on <code>myVar</code>');",
+        "assert(/myVar\\s*\\-{2}/.test(code), 'message: Use the <code>--</code> operator on <code>myVar</code>');",
```

Comment:
I didn't know you could have a space before the decrement operator.  Might as well add it to the increment operator as well.

What I don't understand is why you're escaping `-`?  It's not a special character, I don't think.